api/tests/acceptance/application/authentication/oidc/oidc-route-post_test.js

```diff
@@ -0,0 +1,170 @@
+const { expect, databaseBuilder } = require('../../../../test-helper');
+const createServer = require('../../../../../server');
+const jsonwebtoken = require('jsonwebtoken');
+const authenticationSessionService = require('../../../../../lib/domain/services/authentication/authentication-session-service');
+const { featureToggles } = require('../../../../../lib/config');
+
+describe('Acceptance | Application | Oidc | Routes', function () {
+  describe('POST /api/oidc/token-reconciliation?authentication-key=key', function () {
+    context('when user has no oidc authentication method', function () {
+      it('should return authentication key with 200 HTTP status', async function () {
+        // given
+        databaseBuilder.factory.buildUser.withRawPassword({ email: 'eva.poree@example.net', rawPassword: 'pix123' });
+        await databaseBuilder.commit();
+
+        const server = await createServer();
+        featureToggles.isSsoAccountReconciliationEnabled = true;
+
+        const idToken = jsonwebtoken.sign(
+          {
+            given_name: 'Brice',
+            family_name: 'Glace',
+            nonce: 'nonce',
+            sub: 'some-user-unique-id',
+          },
+          'secret'
+        );
+        const userAuthenticationKey = await authenticationSessionService.save({
+          sessionContent: { idToken },
+          userInfo: {
+            firstName: 'Brice',
+            lastName: 'Glace',
+            nonce: 'nonce',
+            externalIdentityId: 'some-user-unique-id',
+          },
+        });
+
+        // when
+        const response = await server.inject({
+          method: 'POST',
+          url: `/api/oidc/token-reconciliation?authentication-key=${userAuthenticationKey}`,
+          payload: {
+            data: {
+              attributes: {
+                email: 'eva.poree@example.net',
+                password: 'pix123',
+                identity_provider_code: 'CNAV',
+              },
+            },
+          },
+        });
```

Comment:
```suggestion
          url: `/api/oidc/token-reconciliation`,
          payload: {
            data: {
              attributes: {
                email: 'eva.poree@example.net',
                password: 'pix123',
                identity_provider: 'CNAV',
                authentication_key: userAuthenticationKey,
              },
            },
          },
        });
```

Cela s'applique à tous les autres tests